static/app/components/events/interfaces/spans/scrollbarManager.tsx

```diff
@@ -233,11 +237,67 @@ export class Provider extends Component<Props, State> {
   hasInteractiveLayer = (): boolean => !!this.props.interactiveLayerRef.current;
   initialMouseClickX: number | undefined = undefined;
 
-  onScroll = () => {
+  onWheel = (deltaX: number) => {
     if (this.isDragging || !this.hasInteractiveLayer()) {
       return;
     }
 
+    // Setting this here is necessary, since updating the virtual scrollbar position will also trigger the onScroll function
+    this.isWheeling = true;
+
+    if (this.wheelTimeout) {
+      clearTimeout(this.wheelTimeout);
+    }
+
+    this.wheelTimeout = setTimeout(() => (this.isWheeling = false), 200);
```

Comment:
I unfortunately do not think there is any other way to do this aside from using `setTimeout`, since there is no onWheelEnd event or anything like that

Reply:
@0Calories you'll also need to do `this.wheelTimeout = null` within the timeout callback.

Reply:
Oh true 👍